bin/console

```diff
@@ -6,38 +6,43 @@ use Symfony\Component\Console\Input\ArgvInput;
 use Symfony\Component\ErrorHandler\Debug;
 use Symfony\Component\Yaml\Yaml;
 
+if (!in_array(PHP_SAPI, ['cli', 'phpdbg', 'embed'], true)) {
+    echo 'Warning: The console should be invoked via the CLI version of PHP, not the '.PHP_SAPI.' SAPI'.PHP_EOL;
+}
+
 set_time_limit(0);
 
-require __DIR__ . '/../vendor/autoload.php';
-require __DIR__ . '/../src/RequirementChecker.php';
+require dirname(__DIR__) . '/vendor/autoload.php';
 
 $getZikulaEnv = function(ArgvInput $input) {
     $defaultEnv = $input->getParameterOption(['--env', '-e'], getenv('ZIKULA_ENV') ?: 'dev');
     // prefer install and upgrade commands default to prod environment
     $isInstallUpgradeCommand = ((strpos($input->getFirstArgument(), 'zikula:install') !== false)
         || (strpos($input->getFirstArgument(), 'zikula:upgrade') !== false));
     $env = $isInstallUpgradeCommand ? 'prod' : $defaultEnv;
+
     return $env;
 };
 
 $input = new ArgvInput();
-$env = $getZikulaEnv($input);
-$debug = getenv('ZIKULA_DEBUG') !== '0' && !$input->hasParameterOption(['--no-debug', '']) && 'prod' !== $env;
-
-if ($debug) {
-    Debug::enable();
+if (null !== $env = $getZikulaEnv($input)) {
+    putenv('APP_ENV=' . $_SERVER['APP_ENV'] = $_ENV['APP_ENV'] = $env);
 }
 
-// on install or upgrade, check if system requirements are met.
-$kernelConfig = Yaml::parse(file_get_contents(__DIR__ . '/../config/services.yaml'));
-if (is_readable($file = __DIR__ . '/../config/services_custom.yaml')) {
-    $kernelConfig = array_merge($kernelConfig, Yaml::parse(file_get_contents($file)));
+if ($input->hasParameterOption('--no-debug', true)) {
+    putenv('APP_DEBUG=' . $_SERVER['APP_DEBUG'] = $_ENV['APP_DEBUG'] = '0');
 }
-$requirementChecker = new RequirementChecker();
-$requirementChecker->verify($kernelConfig['parameters']);
```

Comment:
please return

Reply:
`console` includes `config/bootstrap.php` which includes and executes the `RequirementChecker`. No need to have it twice.